packages/scripts/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Breaking Changes
 
+-   The bundled `puppeteer-core` dependency has been updated from requiring `^11.0.0` to requiring `^13.1.2` ([#37078](https://github.com/WordPress/gutenberg/pull/37078)).
 -   The bundled `eslint` dependency has been updated from requiring `^7.17.0` to requiring `^8.3.0` ([#36283](https://github.com/WordPress/gutenberg/pull/36283)).
 -   The `puppeteer-core` dependency has been updated from requiring `^10.1.0` to requiring `^11.0.0` (see [Breaking Changes](https://github.com/puppeteer/puppeteer/releases/tag/v11.0.0), [#36040](https://github.com/WordPress/gutenberg/pull/36040)).
```

Comment:
One question: Should I merge those two changelog entries?

Reply:
We should keep the old one, too 👍